Modules/FindTEEV2SDK.cmake

```diff
@@ -0,0 +1,92 @@
+###############################################################################
+# Find Thermal Expert EV2 SDK
+#
+#     find_package(TEEV2SDK)
+#
+# Variables defined by this module:
+#
+#  TEEV2SDK_FOUND                True if EV2 SDK found
+#  TEEV2SDK_VERSION              The version of EV2 SDK
+#  TEEV2SDK_INCLUDE_DIR          The location(s) of EV2 SDK headers
+#  TEEV2SDK_LIBRARY_DIR          Libraries needed to use EV2 SDK
+#  TEEV2SDK_BINARY_DIR           Binaries needed to use EV2 SDK
+
+SET(TEEV2_SDK_PATH_HINTS
+  "../ThirdParty/TE_SDK"
+  "../Plus-bin/TE_SDK"
+  
+  )
+
+
+find_path(TEEV2_SDK_DIR include/i3system_TE.h
+  PATHS ${TEEV2_SDK_PATH_HINTS}
+  DOC "Thermal Expert EV2 SDK directory")
+
+if (TEEV2_SDK_DIR)
+  # Include directories
+  set(TEEV2_SDK_INCLUDE_DIR ${TEEV2_SDK_DIR}/include)
+  mark_as_advanced(TEEV2_SDK_INCLUDE_DIR)
+
+
+  # Libraries
+  SET(PLATFORM_SUFFIX "")
+  SET(BIN_FILENAME "")
```

Comment:
BIN_FILENAME and FILENAME don't seem to be used anywhere.

Reply:
Unused code removed.